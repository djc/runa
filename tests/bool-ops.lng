def __main__(name: str, args: list[str]) -> int:
	print('a' and 'b')
	print('' and 'a')
	print('a' and '')
	print('a' or 'b')
	print('' or 'a')
	print('a' or '')
