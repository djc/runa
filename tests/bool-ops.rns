def main(name: &str, args: &array[str]) -> i32:
	print('a' and 'b')
	print('' and 'a')
	print('a' and '')
	print('a' or 'b')
	print('' or 'a')
	print('a' or '')
	print(True)
	return 0
