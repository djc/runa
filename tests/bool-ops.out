b


a
a
a
True
